### DISCUSSION TITLE
Bootstrap instance kitt4sme.live setup

### DISCUSSION THREAD

**Discussion by edrisimm** [c0c0n3/kitt4sme.live#166] [Q&A]
Hi, when we try to do the bootstrap for the live platform, a problem of "connection refuse" arises. We have been followed all steps (as presented in https://github.com/c0c0n3/kitt4sme.live/blob/main/docs/bootstrap.md) until we try to drop into the Nix shell by performing the "tools.sh" command, as a smoke test (screenshot 1).

We are performing the steps in a VM Linux Ubuntu with system's characteristics as presented in the second screenshot.

Thank you

Enterprise-Wisdom team

![image](https://user-images.githubusercontent.com/100539889/191248233-32ba9e27-9dce-4ec0-8809-e42417257c99.png)

![image](https://user-images.githubusercontent.com/100539889/191248166-47961634-503d-48c6-a1c3-d5110835e13e.png)

**Comment by c0c0n3** [comment]
Hi @edrisimm :-)

Thanks sooo much for reporting this and including enough detail for us to figure out what the problem is. It looks like we've got an issue with our Nix shell whereby we can't build the `kubeseal` utility anymore. I've opened an issue about it, #167, and we'll try to fix it in the next few weeks if possible. In the meantime, if you could please follow the workaround documented in #167 to be able to go on to the next installation step of the bootstrap procedure.
